.github/workflows/ghcr.yml

```diff
@@ -0,0 +1,29 @@
+name: publish to ghcr
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * 1"

```

Comment:
Please remove the cron expression, I would like this to only run on master merges :)